xm-commons-domainevent-db/src/main/java/com/icthh/xm/commons/domainevent/db/service/JpaEntityMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.icthh.xm.commons.domainevent.db.service;
+
+import com.icthh.xm.commons.domainevent.db.domain.JpaEntityContext;
+import com.icthh.xm.commons.domainevent.domain.DomainEvent;
+
+public interface JpaEntityMapper {
```

Comment:
suggest moving to separate package like `com.icthh.xm.commons.domainevent.db.service.mapper`